### ISSUE TITLE
[Legacy] Build Error (branch: master)

### ISSUE THREAD

**Issue by dilijev** [Microsoft/ChakraCore#2318]
https://ci.dot.net/job/Microsoft_ChakraCore/job/master/job/daily_dev12_x64_debug/

```
D:\j\workspace\daily_dev12_x---55cdd258\lib\Jsrt\JsrtHelper.cpp(7): error C2220: warning treated as error - no 'object' file generated [D:\j\workspace\daily_dev12_x---55cdd258\lib\Jsrt\Chakra.Jsrt.vcxproj]
10:57:32 D:\j\workspace\daily_dev12_x---55cdd258\lib\Jsrt\JsrtHelper.cpp(7): warning C4627: '#include <pthread.h>': skipped when looking for precompiled header use [D:\j\workspace\daily_dev12_x---55cdd258\lib\Jsrt\Chakra.Jsrt.vcxproj]
10:57:32           Add directive to 'JsrtPch.h' or rebuild precompiled header
```

Broken since build 35

https://ci.dot.net/job/Microsoft_ChakraCore/job/master/job/daily_dev12_x64_debug/changes

```
35 (Dec 22, 2016 10:57:07 AM)

android: ChakraCore compiles to Android ARMv7 (commit: 6811584) — ogbastem / githubweb
silence failures for connecting to jit process (commit: 1f87d7e) — Michael Holman / githubweb
Resolves #1896. Modified Array.prototype.toLocaleString to use the (commit: 1410874) — akatti / githubweb
AutoRestoreFunctionInfo does not handle throw during initialization (commit: 34f70cd) — Tom Care / githubweb
Update Jsrt Utf8 types and naming (commit: adbea39) — ogbastem / githubweb
```

@obastemur @MikeHolman @atulkatti @tcare

**Comment by Krovatkin** [comment]
:rofl:  I tried to build a master branch today; hit the very same issue. Switched the header order to

```cpp
#include "JsrtPch.h"
#if !defined(_WIN32)
#include <pthread.h>
#endif
```
to only find out afterwards that there is already a PR for this issue. Dang!

**Comment by dilijev** [comment]
Fixed by #2320
